src/xcode/ENA.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,6 +1,15 @@
 {
   "object": {
     "pins": [
+      {
+        "package": "AnyCodable",
+        "repositoryURL": "https://github.com/Flight-School/AnyCodable",
+        "state": {
+          "branch": null,
+          "revision": "b1a7a8a6186f2fcb28f7bda67cfc545de48b3c80",
+          "version": "0.6.2"
+        }
+      },
```

Comment:
I also added this on my branch, is this required to import our package and automatically added?

And I think we need to add the legal reference in the `licenses.json` file, don't we? And to generate the model for the licenses screen?

Reply:
I think this is added because jsonfunctions is using AnyCodable. At that point we don't need to add the license.

Reply:
In other words, we don't need to add licenses for implicit dependencies.